symengine/tests/printing/test_printing.cpp

```diff
@@ -470,3 +476,23 @@ TEST_CASE("Ascii Art", "[basic]")
 {
     std::cout << SymEngine::ascii_art() << std::endl;
 }
+
+TEST_CASE("test_sets(): printing", "[printing]")
+{
+    RCP<const Set> r1, r2;
+    r1 = finiteset({symbol("x"), symbol("y"), symbol("z")});
+    r2 = interval(integer(5), integer(10))->set_complement(r1);
+    REQUIRE(r2->__str__() == "{x, y, z} \\ [5, 10]");
+
+    r1 = finiteset({symbol("x"), integer(7), symbol("z")});
+    r2 = interval(integer(5), integer(10))->set_complement(r1);
+    REQUIRE(r2->__str__() == "{x, z} \\ [5, 10]");
+
+    r1 = finiteset({symbol("x"), symbol("y"), symbol("z")});
+    r2 = interval(integer(5), integer(10));
+    REQUIRE(r1->set_complement(r2)->__str__() == "[5, 10] \\ {x, y, z}");
+
+    r1 = set_complement(interval(NegInf, Inf, true, true),
+                        finiteset({symbol("y")}));
+    REQUIRE(r1->__str__() == "(-oo, oo) \\ {y}");
```

Comment:
Ah. You moved all the printing tests. I meant just 1 as the 4 tests above test the same thing.